### ISSUE TITLE
Implement `eliasfano32.Min` func similar to `eliasfano32.Max`

### ISSUE THREAD

**Issue by AskAlexSharov** [ledgerwatch/erigon#5623]
Target is to get Min without full parsing of eliasfano serialized bytes (ideally: zero-copy, zero-allocation). 
Implementation of Max was trivial, but I don't know if implement Min easy or not. 
If it's not possible or too complicated - let's don't implement it.

**Comment by fenghaojiang** [comment]
Glad to help.

**Comment by fenghaojiang** [comment]
Implement here but still need to parse bytes as input. 
The eliasfano encoding needs to calculate the the lower/upper bits number from `count` and `maxOffset`. Code in `erigon-lib` puts these two numbers into position [:8] and [8:16] and the data will be stored in [16:].

Minimal steps would be like: 
1. parsing [:8] and [8:16], calculate out the `count` and `maxOffset`
2. get the Min data location according to the `count` and `maxOffset`
3. parsing the data location and calculating the target number

https://github.com/ledgerwatch/erigon-lib/pull/682

**Comment by AskAlexSharov** [comment]
@fenghaojiang thank you. 
If you are interesting - I created one more similar issue: https://github.com/ledgerwatch/erigon/issues/5743

**Comment by AskAlexSharov** [comment]
Interesting - can we do that unsafe conversion of mmap without unsafe, by https://go.dev/ref/spec#Conversions_from_slice_to_array_pointer

**Comment by fenghaojiang** [comment]
> Interesting - can we do that unsafe conversion of mmap without unsafe, by https://go.dev/ref/spec#Conversions_from_slice_to_array_pointer

Yep. But it's the same type. Conversion between `byte` and `uint64` doesn't work that way.
